src/data/causeSpecificJSONData/cats.json

```diff
@@ -0,0 +1,55 @@
+{
+  "path": "cats",
+  "data": {
+    "styles": {
+      "colors": {
+        "primary": "FF6A08",
```

Comment:
Include `#` in hex values. It's less logic for the page and should increase flexibility to use rgba.